docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3.4'
```

Comment:
License header missing.

Reply:
Ok! Fixed in 25d79f353727c16f9858d13e157d5867c6e6866a